content/issue-1/minilsm.md

```diff
@@ -0,0 +1,125 @@
+MiniLSM is a working-in-progress tutorial series for building a storage engine. Based on the experience of building
+[AgateDB](https://github.com/tikv/agatedb) and [RisingWave](https://github.com/risingwavelabs/risingwave), MiniLSM
+introduces you the basics of an LSM-tree storage engine with a step-by-step tutorial.
+
+The starter code is accessible on [GitHub](https://github.com/skyzh/mini-lsm), and the tutorial is available on
+[GitHub Pages](https://skyzh.github.io/mini-lsm/).
+
+```urlpreview
+https://github.com/skyzh/mini-lsm
+```
+
+## What is LSM, and Why LSM?
+
+Log-structured merge tree is a data structure to maintain key-value pairs. This data structure is widely used in
+distributed database systems like [TiDB](https://www.pingcap.com) and [CockroachDB](https://www.cockroachlabs.com) as
+their underlying storage engine. [RocksDB](http://rocksdb.org), based on [LevelDB](https://github.com/google/leveldb),
+is an implementation of LSM-Tree storage engine. It provides a wide range of key-value access functionalities and is
+used in a lot of production systems.
+
+Generally speaking, LSM Tree is an append-friendly data structure. It is more intuitive to compare LSM to other
+key-value data structure like RB-Tree and B-Tree. For RB-Tree and B-Tree, all data operations are in-place. That is to
+say, when you update the value corresponding to the key, the value will be overwritten at its original memory or disk
+space. But in an LSM Tree, all write operations, i.e., insertions, updates, deletions, are performed in somewhere else.
```

Comment:
Delete "in" and further clarification on where the write operations are performed and how and why this is important is requested